apps/server/src/modules/room/api/room.uc.ts

```diff
@@ -240,12 +265,31 @@ export class RoomUc {
 
 	public async passOwnership(currentUserId: EntityId, roomId: EntityId, targetUserId: EntityId): Promise<void> {
 		const ownershipContext = await this.getPassOwnershipContext(roomId, currentUserId, targetUserId);
+		const user = await this.authorizationService.getUserWithPermissions(currentUserId);
+
+		const roomMembershipAuthorizable = await this.roomMembershipService.getRoomMembershipAuthorizable(roomId);
+		const roomMembers = await this.roomMembershipService.getRoomMembers(roomId);
+		const roomMember = roomMembers.find((member) => member.userId === targetUserId);
+		const roomOwner = roomMembers.find((member) => member.roomRoleName === RoleName.ROOMOWNER);
+		if (!roomMember) {
+			throw new CantPassOwnershipToUserNotInRoomLoggableException({
```

Comment:
do we really want such a long exception name?
why not `UserNotInRoomLoggableException` and using that for more cases? Or is the destinction important?

Reply:
You are right: it's very long.
On the other hand the chosen name fits better with the other ones:
- CantChangeOwnersRoleLoggableException
- CantPassOwnershipToStudentLoggableException
As the loggable is only used in one place. I think it is okay to have a rather long name.